apps/fluent-tester/src/RNTester/TestComponents/Callout/CalloutTest.tsx

```diff
@@ -97,34 +114,84 @@ export const CalloutTest: React.FunctionComponent<{}> = () => {
+            onValueChange={(color) => setSelectedBackgroundColor(color === colorDefault ? undefined : color)}
+          >
+            {colorSelections.map((color, index) => (
+              <Picker.Item label={color} key={index} value={color} />
+            ))}
+          </Picker>
+
+          <Picker
+            prompt="Border Color"
+            selectedValue={selectedBorderColor || colorDefault}
+            onValueChange={(color) => setSelectedBorderColor(color === colorDefault ? undefined : color)}
+          >
+            {colorSelections.map((color, index) => (
+              <Picker.Item label={color} key={index} value={color} />
+            ))}
+          </Picker>
+
+          <Picker
+            prompt="Border Width"
+            selectedValue={selectedBorderWidth || borderWidthDefault}
+            onValueChange={(width) => setSelectedBorderWidth(width === borderWidthDefault ? undefined : width)}
+          >
+            {borderWidthSelections.map((width, index) => (
+              <Picker.Item label={width.toString()} key={index} value={width} />
+            ))}
+          </Picker>
         </View>
 
         <Separator vertical />
 
-        <View style={{ flexDirection: 'column' }}>
+        <View style={{ flexDirection: 'column', paddingHorizontal: 5 }}>
           <Button content="Press for Callout" onClick={toggleShowStandardCallout} />
           <Text>
             <Text>Visibility: </Text>
             {isStandardCalloutVisible ? <Text style={{ color: 'green' }}>Visible</Text> : <Text style={{ color: 'red' }}>Not Visible</Text>}
           </Text>
         </View>
+
+        <Separator vertical />
+
+        <View style={{ flexDirection: 'column', paddingHorizontal: 5 }}>
+          <View ref={redTargetRef} style={{ height: 20, width: 20, backgroundColor: 'red', padding: 5 }} />
+          <View ref={greenTargetRef} style={{ height: 20, width: 20, backgroundColor: 'green', padding: 5 }} />
+          <View ref={blueTargetRef} style={{ height: 20, width: 20, backgroundColor: 'blue', padding: 5 }} />
+        </View>
       </View>
 
       <Separator />
 
-      <Button componentRef={decoyBtn1Ref} content="Custom reFocus w/ focus in Callout" />
-      <Button componentRef={decoyBtn2Ref} content="Custom reFocus w/o focus in Callout" />
+      <View style={{ paddingVertical: 5 }}>
+        <Button componentRef={decoyBtn1Ref} content="Custom reFocus w/ focus in Callout" />
+        <Button componentRef={decoyBtn2Ref} content="Custom reFocus w/o focus in Callout" />
+      </View>
 
       {showStandardCallout && (
         <Callout
-          target={anchorRef}
-          onDismiss={onDismissStandardCallout}
-          onShow={onShowStandardCallout}
-          onRestoreFocus={customRestoreFocus ? onRestoreFocusStandardCallout : undefined}
-          accessibilityLabel="Standard Callout"
-          setInitialFocus={shouldSetInitialFocus}
+          {...{
```

Comment:
The double curly braces are interesting, might be a little cleaner to create an object and conditionally add in props that may be undefined. Then spread that